### ISSUE TITLE
Configuration error - LP are not working for users 

### ISSUE THREAD

**Issue by apptemplatequestions** [pnp/custom-learning-office-365#461]
Hello,

a customer of mine is having issues with the learning pathways webpart, as they are not loading correctly for the users.

The users get a configuration error displayed, please see the following screenshots below for more details.

![image](https://user-images.githubusercontent.com/69305126/108972097-45df7d80-7683-11eb-846f-2c6486ec89b3.png)

![image](https://user-images.githubusercontent.com/69305126/108971956-221c3780-7683-11eb-943e-f914ea45ed4c.png)

![image](https://user-images.githubusercontent.com/69305126/108971997-2d6f6300-7683-11eb-883a-41d1d8695abc.png)

![image](https://user-images.githubusercontent.com/69305126/108972028-33fdda80-7683-11eb-8a24-08b22cdd08bb.png)

Could you please advise me on how to overcome this issue?

Thanks a lot in advance!

**Comment by newgardj** [comment]
Hello,

Could you confirm the following?

- How was Learning Pathways deployed, which provisioning steps did you use?

- Does it work for Admins of the WorkplaceTraining Site? (Viewer and Admin components?

- What permissions do the users have on the WorkplaceTraining Site?

Your screenshot indicates 33 errors and 7 warnings though we can only see a single warning. The browser trace documentation below can be used to capture data from both the Console and Network messages. Any warnings or failure in both would be helpful in troubleshooting.

https://docs.microsoft.com/en-us/azure/azure-portal/capture-browser-trace

**Comment by karuanag** [comment]
HI @apptemplatequestions Please let us know if you have resolved the issue or are still in need of assistance. Thank you!

**Comment by JamieP91** [comment]
Hi, 

At the company I work for me just provisioned SharePoint Success Site for the Learning Pathways and are getting the same error on any of the pages under Get Started such as Plan Site. 
These are the errors in the console for Chrome. I get similar errors in Edge too.
![image](https://user-images.githubusercontent.com/81352541/112439987-ab399380-8d41-11eb-935a-7ab9d48fb947.png)

We did provision it before updating learning pathways to version 4.1.4 - could this be why? If so, how do we re-provision the SharePoint Success Site?

**Comment by jnewgard-thrive** [comment]
Hi @JamieP91,

While there don't appear to be any Learning Pathways errors in your log, it does suggest that some requests are being blocked by either installed browser extensions or network configuration.

Was Learning Pathways at least version 4.0 before deploying? If so, recreation should not be necessary, however, you can certainly re-provision the site by permanently deleting the existing copy and following the existing steps for provisioning.

[Delete A Site](https://docs.microsoft.com/en-us/sharepoint/delete-site-collection)

The next most likely concern is provisioning the SharePoint Success Site and not provisioning the SharePoint Success Content Pack. This is outlined in step 11 of the setup instructions.

[Provision SharePoint Success Site](https://docs.microsoft.com/en-us/sharepoint/provision-sss#provision-the-sharepoint-success-site-to-your-tenant-from-the-learning-pathways-administrative-page)

If your issue persists, please create a new Issue as this is separate from the initial posted issue.
